tests/Unit/inc/Engine/Admin/Settings/Settings/sanitizeCallback.php

```diff
@@ -19,7 +19,7 @@ class Test_SanitizeCallback extends TestCase {
 	public function setUp() {
 		parent::setUp();
 
-		$this->options  = $this->createMock( Options::class );
+		$this->options  = $this->createMock( Options_Data::class );
```

Comment:
This should be changed to Mockery:mock(), right?

Reply:
Yes it should.

Reply:
https://github.com/wp-media/wp-rocket/pull/2592/commits/3a497ea3a2dcefb601fc4fc0ab9c785ddd8046e7